ros1_bridge/__init__.py

```diff
@@ -114,6 +114,10 @@ def generate_cpp(output_path, template_dir):
     template_file = os.path.join(template_dir, 'get_factory.cpp.em')
     expand_template(template_file, data, os.path.join(output_path, 'get_factory.cpp'))
 
+    data = {'mappings': ordered_mappings}
+    template_file = os.path.join(template_dir, 'get_mappings.cpp.em')
```

Comment:
oops - added in https://github.com/ros2/ros1_bridge/pull/42/commits/e0c68e870878896533b34c7e5396a24a2f1f1d5e